ompi/mpi/man/man3/MPI_Status_f082c.3in

```diff
@@ -0,0 +1,30 @@
+.\" -*- nroff -*-
+.\" Copyright 2013 Los Alamos National Security, LLC. All rights reserved.
+.\" Copyright 2006-2008 Sun Microsystems, Inc.
+.\" Copyright (c) 1996 Thinking Machines Corporation
+.\" Copyright (c) 2020      Research Organization for Information Science
+.\"                         and Technology (RIST).  All rights reserved.
+.\" $COPYRIGHT$
+.TH MPI_Status_f082c 3 "#OMPI_DATE#" "#PACKAGE_VERSION#" "#PACKAGE_NAME#"
+.SH NAME
+\fBMPI_Status_f082c, MPI_Status_c2f08 \fP \- Translates a C status into a Fortran 2008 status, or vice versa.
+
+.SH SYNTAX
+.ft R
+.SH C Syntax
+.nf
+#include <mpi.h>
+int MPI_Status_f082c(const MPI_F08_status \fI*f08_status\fP, MPI_Status \fI*c_status\fP)
+int MPI_Status_c2f08(const MPI_Status \fI*c_status\fP, MPI_F08_status \fI*f08_status\fP)
```

Comment:
It looks like the official bindings for these functions don't actually have `const` in them 😲 : https://lists.mpi-forum.org/pipermail/mpiwg-fortran/2020-September/003586.html

I'm inclined to leave these `const`s here, though (and in the code) because a) they're harmless, and b) I think that this is a mistake that should be fixed in an MPI errata.

Reply:
See mpi-forum/mpi-issues#307.